crates/glue/tests/test_glue_cli.rs

```diff
@@ -190,9 +190,11 @@ mod glue_cli_run {
             ),
         );
 
-        // If there is any stderr, it should be reporting the runtime and that's it!
+        // If there is any stderr, it should only be reporting the runtime
+        // or logging the "rebuilding platform" waiting message.
         if !(glue_out.stderr.is_empty()
-            || glue_out.stderr.starts_with("runtime: ") && glue_out.stderr.ends_with("ms\n"))
+            || glue_out.stderr.starts_with("runtime: ") && glue_out.stderr.ends_with("ms\n")
+            || compile_out.stderr.contains("🔨 Rebuilding platform..."))
```

Comment:
https://github.com/roc-lang/roc/pull/3972/commits/8274bfe9a5e81229f98e1e68bde054678218511b#r967017520 should apply to the 2 places in this file too!